### PR TITLE
Setting up test environment

### PR DESCRIPTION
**Pull request by salindholm**
https://www.pivotaltracker.com/n/projects/2478349
### User Story
```
As a development team
In order to be able to make sure our code is doing what it is supposed to do
We would like to be able to write and run automated tests
```
## Changes proposed in this pull request:
* Configuring Rspec, adding shoulda-matchers, factoryBot and pry-rails

## Packages/Gems added
* Rspec
* Shoulda-matchers
* FactoryBot
* Pry-rails

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+
+RSpec.configure do |config|
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.use_transactional_fixtures = true
+  config.infer_spec_type_from_file_location!
+  config.filter_rails_from_backtrace!
+  config.include FactoryBot::Syntax::Methods
+  config.include(Shoulda::Matchers::ActiveRecord, type: :model)
```
Comment (tochman) [inline spec/rails_helper.rb]: Please remove the parenthesis on this one or add them to the `include` on line 20. Consistency is important.

**Review by tochman — COMMENTED** [review]
I think this is good, have you run `rspec`?

File: spec/rails_helper.rb (hunk 1 above)
Comment (salindholm) [reply spec/rails_helper.rb]: This is now fixed, Rspec is working!

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
```
Comment (tochman) [inline spec/rails_helper.rb]: I wonder about this change.

Comment (salindholm) [reply spec/rails_helper.rb]: It is now changed back to how it was before!

**Pull request merged**